CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ option(CMAKE_VERBOSE_MAKEFILE "Generate verbose Makefiles" OFF)
 include(GNUInstallDirs)
 
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
-find_package(KOKKOS REQUIRED)
+find_package(Kokkos REQUIRED)
```

Comment:
Is the imported target still called `Kokkos::kokkos`?

Reply:
Yessir